src/containers/editor/graph/Toolbar.tsx

```diff
@@ -119,7 +121,7 @@ const ToolbarButton = memo(({ title, onClick, children }: ToolbarButtonProps) =>
   return (
     <Button
       variant="icon"
-      className="h-5 w-5 p-1"
+      className="h-5 w-5 p-0"
```

Comment:
Why do you want to remove the padding inside the button?

Reply:
This is reverted.